### PR TITLE
Set-up

### PR DESCRIPTION
**Pull request by Jordancooke**
- Extracted: step_runner, Glue_runner and DynamoDb_runner
- Adding Serverless.yml, serverless.sh, serverless_remove.yml
- Added Do.sh
- Added configs
- Updated to work with es-results-pipeline quality checks.

### CODE REVIEW

**Review by kingmushroom — APPROVED** [review]

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+flake8==3.7.7
+future==0.18.2
+s3fs==0.4.2
+py4j==0.10.7
+pyarrow==0.16.0
+awswrangler==1.0.4
+jsonschema==3.2.0
+yamllint==1.20.0
+git+https://github.com/ONSdigital/es-functions.git
+git+https://github.com/ONSdigital/spp-logger
```
Comment (Piwington) [inline dev-requirements.txt:20]: Can this please be Alphabetical.

File: gluerunner/gluerunner.py [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+# merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+# INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
```
Comment (Piwington) [inline gluerunner/gluerunner.py:15]: Can we remove this comment.

File: steprunner/steprunner.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
+# coding=utf-8
+from __future__ import unicode_literals
```
Comment (Piwington) [inline steprunner/steprunner.py:2]: Where is this used I can't see it in the code. Does it does something not obvious?

**Review by Piwington — DISMISSED** [review]

File: gluerunner/gluerunner.py (hunk 2 above)
Comment (Jordancooke) [reply gluerunner/gluerunner.py:15]: Removed

File: steprunner/steprunner.py (hunk 3 above)
Comment (Piwington) [reply steprunner/steprunner.py:2]: At Mike's suggestion tested the removal of this in Sandbox. Steprunner ran successfully so we can remove this.

**Review by rhodriguerrier — APPROVED** [review]

**Review by glanvl — APPROVED** [review]

**Pull request merged**